src/components/Date/Date.tsx

```diff
@@ -0,0 +1,35 @@
+import "./Date.css";
+
+const MonthAndDayDate = ({
+  date,
+  size,
+}: {
+  date: Date;
+  size: "small" | "medium" | "large";
+}) => {
+  const monthNames = [
+    "January",
+    "February",
+    "March",
+    "April",
+    "May",
+    "June",
+    "July",
+    "August",
+    "September",
+    "October",
+    "November",
+    "December",
+  ];
+  return (
+    <div className={`flex justify-center pt-4 ${size}`}>
+      <p className="mr-1 calendar"> 🗓️ </p>
+      <div className="flex flex-col">
+        <p className="mt-1 date"> {date.getDate()}</p>
+        <p> {monthNames[date.getMonth()]}</p>
```

Comment:
I would use `<time>` elements here since both of them refer to "date"-related information: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time.